codec-http2/src/main/java/io/netty/handler/codec/http2/Http2RemoteFlowController.java

```diff
@@ -67,11 +67,11 @@
     /**
      * Determine if the {@code stream} has bytes remaining for use in the flow control window.
      * <p>
-     * Note that this only takes into account HTTP/2 flow control. It does <strong>not</strong> take into account
-     * the underlying {@link io.netty.channel.Channel#isWritable()}.
+     * Note that it will take into account channel writability, and so will return {@code false} if
```

Comment:
`Note that it will take` -> `Note that this method respects channel writability. The channel must be writable for this method to return {@code true}.`